README.md

```diff
@@ -263,6 +264,24 @@ End-to-end Test Service (ETS) for Wire's test automation suite.
 ]
 ```
 
+### `POST /api/v1/instance/<instanceId>/markEphemeralRead`
+
+```json
+{
+  "conversationId": "<string in UUID format>",
+  "messageId": "<string in UUID format>",
+  "userIds": "[<string>]"
+}
+```
+
+#### Response
+
+```json
+{
+  "conversationId": "<string in UUID format>"
```

Comment:
The implementation returns instanceId and instance name. Not sure what is right...